digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -765,7 +766,8 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
       "endMValue" -> roadAddressLink.endMValue,
       "sideCode" -> roadAddressLink.sideCode.value,
       "linkType" -> roadAddressLink.linkType.value,
-      "roadLinkSource" -> roadAddressLink.roadLinkSource.value
+      "roadLinkSource" -> roadAddressLink.roadLinkSource.value,
+      "blackUnderline" -> roadAddressLink.blackUnderline
```

Comment:
wouldn't it be sufficient to have "blackUnderline" -> roadAddressLink.roadType.value==3  <ofc 3 being the variable but you get the idea>?

Reply:
was it because last line in the story?  I think it was written  just so it would be easier to implement. Thus "future story remark".

Reply:
No it would not, in the bottom of the user story it states that "For now, the styling should use stroke only in the situation where 50% or over of the total length of the link has ROAD_TYPE value 3.
" so with that in mind I made that precise check on the backend and condensed the result to a single boolean value (because evaluating if the total geometry length of the parts that have 3 as the road type is >= 50% of the total geometry of the road in the front end would just be a hog of resources) therefore the parameter is passed so that the styler can pick it up with ease.